src/composables/useFormSpaceSettings.ts

```diff
@@ -33,6 +33,10 @@ const EMPTY_SPACE_FORM = {
     type: '',
     privacy: ''
   },
+  boost: {
+    enabled: false,
```

Comment:
will be enabled by default no?

```suggestion
    enabled: true,
```